cursorless-talon-dev/src/cursorless_test.talon

```diff
@@ -9,8 +9,12 @@ test api command bring <user.cursorless_target>:
     user.cursorless_command("replaceWithTarget", cursorless_target)
 test api get text <user.cursorless_target>:
     user.cursorless_get_text(cursorless_target)
-test api get text list <user.cursorless_target>:
+test api get text list on <user.cursorless_target>:
     user.cursorless_get_text_list(cursorless_target)
+test api get text hide decorations <user.cursorless_target>:
+    user.cursorless_get_text(cursorless_target, 1)
```

Comment:
I would prefer to use `true` instead of `1`

Reply:
oh is that a thing now? i have a vague recollection of that. It's lowercase?

Reply:
Yes